lib/exec.js

```diff
@@ -24,9 +24,9 @@
 
 import child_process from 'child_process';
 import fs from 'fs';
-import path from 'path';
 
 import Graceful from 'node-graceful';
+import path from 'path';
```

Comment:
not sure why this was moved, but it belongs in the 'builtin' import group above.

Reply:
Argh stupid autoformat